src/axom/core/execution/execution_space.hpp

```diff
@@ -95,7 +95,7 @@ struct execution_space
   #include "axom/core/execution/internal/omp_exec.hpp"
 #endif
 
-#if defined(AXOM_USE_CUDA) && defined(AXOM_USE_RAJA) && defined(AXOM_USE_UMPIRE)
+#if defined(AXOM_USE_CUDA) && defined(AXOM_USE_RAJA) && defined(AXOM_USE_UMPIRE) && defined(__CUDACC__)
```

Comment:
Should this be (AXOM_USE_RAJA || AXOM_USE_UMPIRE)? Or, are they always used together or not at all?

Reply:
The file it is including has unguarded includes of both RAJA and Umpire.  I cannot speak to whether that is correct just that it is doing it.